azure/template.json

```diff
@@ -266,11 +266,11 @@
                                 "value": "[reference('storage-account').outputs.storageConnectionString.value]"
                             },
                             {
-                                "name": "AzureWebJobsStorage",
-                                "value": "[reference('storage-account').outputs.storageConnectionString.value]"
+                                "name": "ServiceBusConnectionString",
+                                "value": "[reference('service-bus').outputs.ServiceBusEndpoint.value]"
                             },
                             {
-                                "name": "AzureWebJobsDashboard",
```

Comment:
I would probably leave this one in there just.

Reply:
Thanks, AzureWebJobsDashboard put back as suggested.